function-sidecar/cmd/function-sidecar.go

```diff
@@ -94,7 +95,7 @@ func main() {
 		output = ""
 	}
 
-	log.Printf("Sidecar for function '%v' (%v->%v) using %v dispatcher starting\n", group, input, output, protocol)
+	log.Printf("Sidecar for function '%v' (%v->%v) using %v on port %v dispatcher starting\n", group, input, output, protocol, port)
```

Comment:
The sentence now doesn't make sense (word "dispatcher" needs to be next to the protocol)